pike/test/__init__.py

```diff
@@ -167,8 +168,8 @@ def assert_error(self, status):
 
         try:
             yield o
-        except model.ResponseError as e:
-            pass
+        except model.ResponseError as err:
+            e = err
```

Comment:
Consider combining with the chunk below and using `raise_from`.

Reply:
lol, yeah. for sure. not sure what futurize had in mind here

Reply:
I think `e` goes out of scope after the `except` now.